dependencies/ACE/config.h.in

```diff
@@ -14,9 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+ 
 #define ACE_LACKS_IOSTREAM_TOTALLY 1
 #define ACE_HAS_VERSIONED_NAMESPACE 1
 
+#if __cplusplus >= 201103L
+// Force ACE to use C++11 since it fails to autodetect in all cases.
```

Comment:
Hmm.. seems odd

Reply:
Yeah, and it breaks Linux. I’m investigating.